lib/moon_web/components/props_table.ex

```diff
@@ -68,6 +68,10 @@ defmodule MoonWeb.Components.PropsTable do
             :description => prop[:doc] || "-"
           }
         end)))
-    |> Enum.filter(&(!String.starts_with?(&1[:description], "Experimental:")))
+    # |> Enum.filter(&(!String.starts_with?(&1[:description], "Experimental:")))
```

Comment:
just remove commented line pls

Reply:
Since test was failing, I made an additional fix, please take a look at the latest change.